packages/devkit/src/integrationmodel.js

```diff
@@ -650,15 +650,26 @@ export default class IntegrationModel {
    * Remove events to formulas in the DOM target.
    */
   removeEvents() {
-    const eventTarget = this.isIframe ? this.target.contentWindow.document : this.target;
+    const eventTarget = this.isIframe && this.target.contentWindow?.document ? this.target.contentWindow.document : this.target;
+    if (!eventTarget) {
+      return;
+    }
```

Comment:
```suggestion
    const eventTarget = this.isIframe && this.target.contentWindow?.document ? this.target.contentWindow.document : this.target;
    
    if (!eventTarget) {
      return;
    }
    
```

Please let's wrap code blocks with empty lines so it becomes a little bit more readable.